src/attributes.jl

```diff
@@ -910,6 +910,31 @@ A model attribute for the number of results available.
 """
 struct ResultCount <: AbstractModelAttribute end
 
+"""
+    ConflictStatus(result_index)
+    ConflictStatus()
+
+A model attribute for the `TerminationStatusCode` explaining why the conflict
+refiner stopped when computing the conflict `result_index`.
+
+If a minimal conflict is found, it will return `OPTIMAL`.
+If the problem is feasible, it will return `INFEASIBLE`.
+If `compute_conflict` has not been called yet, it will return
+`OPTIMIZE_NOT_CALLED`.
```

Comment:
It might be better to create a new enum with three entries. `OPTIMIZE_NOT_CALLED` and `INFEASIBLE` are confusing here.